index.html

```diff
@@ -3,6 +3,8 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
```

Comment:
You may want to save this snippet somewhere, as you'll need it a lot in the future.